app/helpers/moments_helper.rb

```diff
@@ -21,22 +21,17 @@ def edit_moment_props(moment, viewers)
     )
   end
 
-  # rubocop:disable MethodLength
   def moments_stats
-    total_count = current_user.moments.all.count
-    monthly_count = current_user.moments.where(
-      created_at: Time.current.beginning_of_month..Time.current
-    ).count
-    return '' if total_count <= 1
+    return '' if moment_count[:total] <= 1
```

Comment:
@hmasila I noticed this line returns `""` when a user has <= 1 moments. which makes Line_170 redundant. Cc @julianguyen

Reply:
@bmwachajr https://github.com/ifmeorg/ifme/pull/1366/files#diff-5f72a77ce1e4b1289dbf5eb071fb44f1R160 uses `moment_count[:total]` which is totally fine because it doesn't need this method